server/schema/dashboard.go

```diff
@@ -75,16 +78,22 @@ const (
 	DashboardElementTypeReportTemplate DashboardElementType = "report"
 	DashboardElementTypeGARealtime     DashboardElementType = "ga-realtime"
 	DashboardElementTypeNRServers      DashboardElementType = "nr-servers"
+	DashboardElementTypeTCBuilds       DashboardElementType = "tc-builds"
 )
 
 func (t DashboardElementType) Clean() DashboardElementType {
-	switch t {
-	case DashboardElementTypeReportTemplate, DashboardElementTypeGARealtime,
-		DashboardElementTypeNRServers:
+	validValues := map[DashboardElementType]bool{
+		DashboardElementTypeReportTemplate: true,
+		DashboardElementTypeGARealtime:     true,
+		DashboardElementTypeNRServers:      true,
```

Comment:
Of cause, `switch` is faster as well as takes less memory, but as I've notice in the commit message, I really believe that according to some metrics dashboard, which is actually not under big stress, readability is much more important that small efficiency reduce.